bootstrap-salt.sh

```diff
@@ -4325,6 +4353,179 @@ install_freebsd_restart_daemons() {
 
 #######################################################################################################################
 #
+#   OpenBSD Install Functions
+#
+
+install_openbsd_deps() {
+  ping -c 2 -q mirrors.nycbug.org || return 1
+  echo "installpath = http://mirrors.nycbug.org/pub/OpenBSD/${OS_VERSION}/packages/${CPU_ARCH_L}/" >/etc/pkg.conf || return 1
```

Comment:
Is it possible to reference all mirrors in a generic manner?  If not, I guess this will be fine.